evento/src/components/TabContainer/index.js

```diff
@@ -1,23 +1,37 @@
 import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
 import './TabContainer.css';
 
-const Tab = (props) => {
+const Tab = ({title, onClick, isSelected}) => {
```

Comment:
Imo it is clearer to use props instead of destructuring (clearer distinction between props and local variables), but if you prefer that then no need to change